### DISCUSSION TITLE
cfg files

### DISCUSSION THREAD

**Discussion by kotovasia5120** [kachurovskiy/nanoels#230] [Ideas]
I would like all the settings to be stored in files for easy updating and configuration.
For example:
buttons.cfg
pinout.cfg
axis.cfg
json?
I'll try to do it myself, but I'm giving you an idea.

**Comment by kachurovskiy** [comment]
@kotovasia5120 sure, whatever works best for you.

Overall, everything is purposely in 1 file for a number of reasons e.g. simplicity, easier for people to share their setup when something isn't working, etc.

**Reply by kotovasia5120** [reply]
It's just that some settings are scattered throughout the code. With files, it would be more centralized.
